package.json

```diff
@@ -33,14 +36,16 @@
     "prettier": "^2.8.7",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "rollup-plugin-cleanup": "^3.2.1",
     "sass": "^1.60.0",
     "stylelint": "^15.3.0",
     "stylelint-config-standard-scss": "^7.0.1",
     "tslib": "^2.4.0",
     "typescript": "^5.0.3"
   },
   "workspaces": [
-    "packages/*",
+    "packages/utils",
+    "packages/core-ui",
```

Comment:
Is the wildcard not valid in this case? Or are you just leaning on explicitly listing the packages as preferred?

Reply:
Wild card is valid. The important thing to know though is to have an order, as the commands with --workspace will execute on that order. So its better to keep an order